Samples/kmp-app/shared/src/commonMain/kotlin/sample/kpm_app/SharedBusinessLogic.kt

```diff
@@ -0,0 +1,26 @@
+package sample.kpm_app
+
+import io.sentry.kotlin.multiplatform.SentryKMP
+
+object SharedBusinessLogic {
```

Comment:
Any reason why not to use the SentryKMP directly?

Reply:
this is just for a sample that it also works in the shared layer as well as directly in the platform code